tracer/src/Datadog.Trace/Headers/Ip/IpExtractor.cs

```diff
@@ -0,0 +1,141 @@
+// <copyright file="IpExtractor.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
```

Comment:
For other reviewers, I believe the only part that have changed and thus to be reviewed starts line 104 (IsPrivateIp method)

Reply:
Yes mostly to take into account IsIPv6UniqueLocal, I changed the location of it cause it's more of a tracer span tag rather than a security one now. And the ip collection algorithm is slightly changed, collecting only the first private ip found